.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Crate Pull Request Form
```

Comment:
This seems to still be named "Crate". I'm not sure if this template is still in use, but this would be a quick fix if it is!

Reply:
Good catch Sienna! I'll change that real quick.